manage.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python
```

Comment:
Je tento soubor vůbec potřeba? Testy běží i bez něj a projekt není dostatečně zkonfigurován aby běžel nějaký example přes `runserver`.

Reply:
Tady je hlavně pro to aby se dali dělat `makemigrations`